qutip/utilities.py

```diff
@@ -404,6 +404,12 @@ def iterated_fit(
         `[[p11, ..., p1n], [p21, ..., p2n], ..., [pN1, ..., pNn]]`.
     """
 
+    # Check if array is constant
+    # if (ydata == ydata[0]).all():
+    #     if num_params == 3:
+    #         return [ydata[0]], [0], [0]
+    #     else:
+    #         return [ydata[0]], [0], [0], [0]
```

Comment:
What is this doing here?

Reply:
Sorry, Forgot to delete the comment 😞  , will delete in the next commit